internal/transport/controlbuf.go

```diff
@@ -527,12 +530,16 @@ const minBatchSize = 1000
 // As an optimization, to increase the batch size for each flush, loopy yields the processor, once
 // if the batch size is too low to give stream goroutines a chance to fill it up.
 func (l *loopyWriter) run() (err error) {
```

Comment:
I had removed it, but added it back in to get the logging inside the `defer`.  Since there are many different return paths, I didn't want to put that logging everywhere.